go.mod

```diff
@@ -137,5 +137,6 @@ require (
 
 replace (
 	github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
+	github.com/openshift/api => /Users/mustafa/workspace/go/src/openshift/api
```

Comment:
This is just till https://github.com/openshift/api/pull/1736 merges 

I had to use the field from the CR to proceed